internal/db/models/project.go

```
@@ -0,0 +1,18 @@
+package models
+
+import (
+	"time"
+
+	"gorm.io/gorm"
+)
+
+type Project struct {
```

golangci-lint (1.64.8)

[warning] exported: exported type Project should have comment or be unexported

(revive)